javascript/new-src/Gruntfile.js

```diff
@@ -1,4 +1,39 @@
 /*jshint node: true*/
+// array of source files to concat
+var targets = {
+  js: [
```

Comment:
Could this be js_src or something perhaps, so it looks more obvious below in the concats?